backend/api/communication/src/main/java/co/airy/core/api/communication/ConversationsController.java

```diff
@@ -75,8 +80,8 @@
                 ConversationListResponsePayload.builder()
                         .data(response)
                         .responseMetadata(
-                                ConversationListResponsePayload.ResponseMetadata.builder()
-                                        .filteredTotal(filteredTotal)
+                                ResponseMetadata.builder()
+                                        .filteredTotal(conversations.size())
```

Comment:
This is a bit counterintuitive: For Lucene we get all the filtered documents returned (`conversations.size()`) and paginate on them. For the rocksdb version, the filtered total is equal to the total, which is also `conversations.size()`